rfcs/054-prismic-api/README.md

```diff
@@ -0,0 +1,110 @@
+- We shouldn't create new IDs for exhibitions, events, stories, etc. There aren't any situations where we should need to merge content from multiple sources, so we should use the document IDs directly from prismic.
+- The elasticsearch index mapping should represent the contract between the pipeline and the API. The API shouldn't need to know anything about the structure of the data in prismic.
+
+## Proposed endpoints
+
+### `/articles`
+
+Filter for type eg stories, webcomics
+
+#### Different kinds of thing (are these 'labels'?)
+
+- story <https://wellcomecollection.org/articles/Y9vktREAACYA1v2U> at /articles
+- webcomic <https://wellcomecollection.org/articles/Y86rshEAACcAI4g4> at /articles
+- book <https://wellcomecollection.org/books/Y7QWrREAAHC43oH0> at /books
+- series <https://wellcomecollection.org/series/WfpEBCoAACrdd43y> composed of articles
+- in pictures <https://wellcomecollection.org/articles/Y-oAIhAAACAAix32> at /articles
+
+#### Filters
+
+- date
+- contributor
+- series
+- contentType
+
+#### Sort
+
+- date
+- relevance
+
+### `/exhibitions`
+
+#### Filter
+
+- date
+- status
+- location
+- contributor?
+- format (ie permanent, exhibition, installation)
+- season
+
+#### Sort
+
+- date
+- relevance
+
+### `/events`
+
+#### Filter
+
+- date
+- accessibility status (is it audio described, signed, etc)
+- location
+- format (eg discussion)
+- event-series
+- season
+- audience (eg schools)
+- contributor
+- isOnline?
+- availableOnline?
+
+#### Sort
+
+- date
+- relevance
+
+### `/books`
+
+?????????
+
+## Open questions
+
+- How do we represent functional content?
+- Should we include the full prismic API response at eg `/articles/{id}`?
+    - should we be able to list all of the docs of a given type at once? what should that url look like?
+- If we want to avoid maintaining multiple versions of the same code, should this work be done as an extension to the concepts API?
+- Works exist at [api.wellcomecollection.org/catalogue/v2/works](api.wellcomecollection.org/catalogue/v2/works), concepts at [/catalogue/v2/concepts](api.wellcomecollection.org/catalogue/v2/concepts), images at [/catalogue/v2/images](api.wellcomecollection.org/catalogue/v2/images).  
+Stories etc aren't part of the catalogue - should the URL be different?
+- what are the top level types? is it articles? stories? series?
+- where do books go? they appear on the stories landing page, so should they appear in stories search results? or are they at a separate endpoint?
+- what is a serial? what is a series?
+- is 'In pictures' the same as 'Image gallery'?
+- Should we be able to filter by contributor role? is that something for the first pass or something we address later? which contributor roles are there? yes for stories? no for exhibitions and events?
+- can we filter exhibitions for accessibility, ie whether a digital guide exists? we can for events, but unclear for exhibitions
```

Comment:
@rcantin-w to investigate

Reply:
Right, so exhibition content types in Prismic do offer a BSL information and and Audio description information field, but after talking with Danny, I can confirm they are not being used (and not displayed when filled in; should we delete them?).

I know we'd thought about it, but as Exhibition Guides should be for every exhibition moving forward, he doesn't think it'd be a valuable filter. 

We might want to consider BSL tours and Dimmed lights times as more of events anyway, so adding an Accessibility filter to Exhibition doesn't seem necessary.

Loved the idea for events though 👍 

cc @harrisonpim

Reply:
Great, thanks @rcantin-w!

Even if accessibility info isn't a meaningful filter for exhibitions, do you think we should we aim to include a link to the digital guide in the API response? My instinct is no, given this goal

> The API should only return enough information for users to figure out whether a result is relevant to them

but I'm open to other arguments!

Reply:
The only reason we'd want that would be for a quick access, which I'm thinking we can determine later on if needed, based on the amount of people accessing it from the exhibition page itself?